### ISSUE TITLE
Undersized touch targets on advanced search (expand/collapse toggles)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2394]
Across Center Row, Cozy, Papers, Thanks, Roy, The Daily, and Default, expand / collapse toggles within advanced resource search forms are not  quite meeting the 24 x 24 px min standard for touch targets. They're mostly in the ballpark of ~6 x 19.5px, though this varies across the stated themes.

**Comment by kimisgold** [comment]
All these themes should now have corrected touch target sizes and are ready for testing.

**Comment by aviannamiller** [comment]
On Center Row, toggles are 24x22px. I'm also not seeing a change for Default.

**Comment by kimisgold** [comment]
What are you seeing for Default and on what browser? Center Row should have a new fix.

**Comment by aviannamiller** [comment]
Center Row looks good. For Default, I'm seeing the following on Chrome, Firefox, Safari:
<img width="258" height="83" alt="Expand toggle. Touch target size is displayed as 7.5 x 24 px" src="https://github.com/user-attachments/assets/e9f5ceee-1e6d-4225-8372-0cd27de6daf9" /> <img width="274" height="83" alt="Collapse toggle. Touch target size is displayed as 12.5 x 24 px" src="https://github.com/user-attachments/assets/db4f630f-d4e0-4fbf-be81-277d7e69ca72" />

**Comment by kimisgold** [comment]
Just pushed a possible fix on Default that moves the min height and width out of the pseudo element and applies them to the anchors themselves.
